src/block/icon-label/editor.scss

```diff
@@ -1,9 +1,9 @@
 .stk-block-icon-label [data-type="stackable/icon"] {
-	flex: 0 0 64px;
+	flex: 0 0 0;
 }
 // Icon label can collapse in mobile in the editor.
 .stk-preview-device-mobile .stk-block-icon-label .block-editor-block-list__layout [data-type="stackable/icon"] {
-	flex: 0 0 64px;
+	flex: 0 0 0;
```

Comment:
Does it work if we remove the flex definition completely since this is for the old `0 0 64px`?

Reply:
it still needs the flex, but i changed it to `flex: 0` instead.